docs/source/manim_directive.py

```diff
@@ -178,6 +179,7 @@ def run(self):
             f'file_writer_config["text_dir"] = r"{text_dir}"',
             f'file_writer_config["video_dir"] = r"{video_dir}"',
             f'file_writer_config["save_last_frame"] = {save_last_frame}',
+            f'file_writer_config["write_to_movie"] = {write_to_movie}',
```

Comment:
@behackl I assume you reviewed this and approved it?

Reply:
I ask because I almost miss it since this PR was meant to be only about examples

Reply:
Yes, it's why I asked for clarification on it: https://github.com/ManimCommunity/manim/pull/576#pullrequestreview-514819023

I can understand the reason behind them, but I also agree that it's not "best practice" to mix a PR adding examples with these kind of changes.

Reply:
Ah thank you, missed that, sorry!

I vote we revert these changes on this PR. If @MysaaJava you have added an animation that is actually breaking the docs build, then please:
1. remove that example from this PR
2. open a separate issue reporting it
3. open a separate PR fixing it
4. add that example animation to the examples gallery after all the above are done

Sorry to be fussy about this, but the FileWriter is *very finicky*. It's possibly the class that has given us the most problems, above everything else.

Reply:
@MysaaJava, would it be ok for you to remove the scenewriter changes from this pr and move it to another pr? 
This would be a bit of extra work, however, I agree that this is the cleaner approach.

Reply:
@leotrs : The error can be easily fixed: It is "FileNotFoundError: [Errno 2] No such file or directory: '../logo/cropped.png'" so only the address of the logo should be changed in that example. Therefore, step 1-4 are not necessary for my opinion.

Reply:
> so only the address of the logo should be changed in that example

I am not particularly fond of fixing this by just adjusting the location. I am not confident enough that the process will actually *always* be called from `docs/source`. See 
https://github.com/ManimCommunity/manim/pull/576#discussion_r510819276

Reply:
ok, I see, thanks for pointing this out.

Reply:
An idea i had was to pass the manim directory location to the manim piece of code (in the manim_directive.py). I'll split the PRs tomorrow.

Reply:
Maybe the problem I have with this example is something else: I'd like for users to be able to copy&paste any example, and it should yield the same result as in the docs.

With examples that include local file paths, that's a difficult thing, and that is probably why I'd rather like to see this example gone.

Reply:
I agree that the examples should be copy-pastable, but this example, i think, will be massively used and therefore it _has_ to be in the docs. What do you think of adding a file_writer_config["manim_location"] (which we can get from python through the module if i remember correctly). and then using this in the example (it will not depend on the working directory of the make command). And of course, add a comment above the line in the example saying «Here put your image file path»

Reply:
> I agree that the examples should be copy-pastable, but this example, i think, will be massively used and therefore it has to be in the docs.

Yes. That's why we want this to be copy-pastable. Because it will be massively used.

Reply:
The only way to make it copy-pastable is to use an online image like it was before. But

- It means you have to have an internet connection to build the docs and to use you pasted code
- If the link destination changes (like recently with the logo file changing name), you will not detect it on github checks (because the commit will not be on master when the checks is run).
- Because of that, you cannot build the docs any version that was before the logo change, beccause the old logo file is not here anymore
- I don't think the `requests.get("https://raw.githubusercontent.com/ManimCommunity/manim/master/logo/cropped.png",stream=True).raw)` code is really readable for newcommers and they will probably never use that (they will use files on their computer, not on the internet).

Reply:
I dislike both variants, but if I had to choose, I'd go with requiring an internet connection.

Reply:
I removed the example for now, i will try to find something better in the future